.travis.yml

```diff
@@ -39,6 +39,12 @@ addons:
   # make sure simplehttp simple verification works (custom /etc/hosts)
   hosts:
     - le.wtf
+    # For boulder-integration.sh
```

Comment:
This comment is misleading - `le.wtf` from above is also used in `boulder-inteagration.sh` - i would recommend to amend comment about `simplehttp` from above and add info that it also necessary for simple_verify for `http-01` and `tls-sni-01` (BTW, it shouldn't be called "simplehttp" any more).

I believe there should be a doc for developers that needs to be updated with the list of necessary mocked domains, but I don't see it being updated in this PR.